2022-03-13 17:22:58,083 DEBUG    debug
2022-03-13 17:22:58,084 INFO     info
2022-03-13 17:22:58,084 WARNING  warning
2022-03-13 17:22:58,084 ERROR    error
2022-03-13 17:22:58,084 CRITICAL critical
2022-03-13 18:35:09,119 DEBUG    Client app started with args: ['/media/sf_selflearning/hostclientapps/HW/lesson5/client.py']
2022-03-13 18:35:09,120 CRITICAL Can't connect to 127.0.0.1:7777: [Errno 111] Connection refused
2022-03-13 19:31:34,253 DEBUG    Client app started with args: ['/media/sf_selflearning/hostclientapps/HW/lesson5/client.py']
2022-03-13 19:31:34,253 INFO     Connected to server address: 127.0.0.1:7777
2022-03-13 19:31:34,254 DEBUG    Creating presence message from Guest
2022-03-13 19:31:34,255 INFO     Response status OK
2022-03-13 19:31:34,255 DEBUG    Response processing result: 200: OK
2022-03-20 20:43:08,021 DEBUG    Client app started with args: ['/media/sf_selflearning/hostclientapps/HW/lesson6/client.py']
2022-03-20 20:43:08,021 INFO     Connected to server address: 127.0.0.1:7777
2022-03-20 20:43:08,023 DEBUG    Creating presence message from Guest
2022-03-20 20:43:08,040 DEBUG    Calling [post_data] from function [main] with args: ({'action': 'presence', 'user': {'account_name': 'Guest'}, 'time': 1647790988.0407646}, <socket.socket fd=5, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 53388), raddr=('127.0.0.1', 7777)>); kwargs: {}
2022-03-20 20:43:08,051 DEBUG    [common.utils.post_data] successfully completed. Returned results: None
2022-03-20 20:43:08,062 DEBUG    Calling [get_data] from function [main] with args: (<socket.socket fd=5, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 53388), raddr=('127.0.0.1', 7777)>,); kwargs: {}
2022-03-20 20:43:08,063 DEBUG    [common.utils.get_data] successfully completed. Returned results: {'response': 200, 'time': 1647790988.0528038, 'alert': 'OK'}
2022-03-20 20:43:08,063 INFO     Response status OK
2022-03-20 20:43:08,063 DEBUG    Response processing result: 200: OK
